tools/circt-test/circt-test.cpp

```diff
@@ -66,6 +69,12 @@ Options opts;
 // Tool Implementation
 //===----------------------------------------------------------------------===//
 
+bool shouldIgnore(verif::FormalOp formalOp, Options &opts) {
+  return formalOp->hasAttr("ignore") &&
```

Comment:
nit: You can use hasAttrOfType to safely cast the attribute
```suggestion
  return formalOp->hasAttrOfType<BoolAttr>("ignore") &&
```